boards/arm/reel_board/doc/index.rst

```diff
@@ -62,6 +62,20 @@ which can be used as an interrupt source.
 
      reel board back (Credit: PHYTEC)
 
+.. note::
+   From summer 2019, the boards are equipped with a new display E0213A22.
```

Comment:
In the ``reel_board/Kconfig.board`` file you call it an ``GDEH0213B72 display`` but you call it ``E0213A22`` here.  I have a reel board where the printing on the ribbon cable says ``HINK-E0213-G01 Date: 2015002-12``.  I presume this is the original reel board but doesn't match either of your display IDs in the Kconfig.board file.